test/compflags/kushal/llvm-incremental.bad

```diff
@@ -0,0 +1 @@
+Incremental compilation is not yet supported with LLVM
```

Comment:
Correction: the expected output is:

error: Incremental compilation is not yet supported with LLVM